src/components/Contentful/DynamicHero.vue

```diff
@@ -97,9 +95,7 @@
 <script>
 import KvButton from '@/components/Kv/KvButton';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
-import KvCarousel from '@/components/Kv/KvCarousel';
```

Comment:
I think all of these imports can be removed, and you can set them as dynamic imports directly in the components object like
```
  components: {
    KvCarousel: () => import('@/components/Kv/KvCarousel')
    KvCarouselSlide: () => import('@/components/Kv/KvCarouselSlide')
    ...
  }
```

And then just use them regularly in the template without needing the data or `<component is="...>` stuff

https://vuejs.org/v2/guide/components-dynamic-async.html#Async-Components

I think we'd have to run the webpack bundle analyzer thing to verify. Or maybe we can just watch the network tab to see JS chunks come in?

Reply:
@emuvente Did you have a good testing method you used when doing the homepage dynamic import stuff?

Reply:
Yeah I wasn't unsure if these 2 approaches are the same

Reply:
Changed it to @ryan-ludwig suggestion above, since it's definitely more readable than `component is`